lib/ash_authentication/verifier.ex

```diff
@@ -139,4 +151,18 @@ defmodule AshAuthentication.Verifier do
       :ok
     end
   end
+
+  # I'm sure this has already been done somewhere else in the Sparkiverse,
+  # so maybe this can be removed and replaced by whatever Spark usually uses
+  # when logging module names?
+  defp bad_token_resource_to_string(module) when is_atom(module) do
+    module
+    |> to_string()
+    |> String.split(".")
+    |> tl()
+    |> Enum.join(".")
+  end
```

Comment:
What about this logic here, @zachdaniel? I know you're logging module names so I assume that you're doing something similar somewhere in Spark?

Reply:
oh, right, all ou need here is `inspect(module)`

Reply:
> oh, right, all ou need here is `inspect(module)`

Now I feel very silly, indeed!

Fixed, though!